scripts/sql/xavier-changes-2016_April/aatamsDB_queries_#268.sql

```diff
@@ -0,0 +1,53 @@
+﻿set search_path = aatams, public;
+
+begin;
+
+-- 1. Clean up animal_release table by deleting animal_release records with no corresponding surgery
+DELETE FROM animal_release ar
+WHERE NOT EXISTS (SELECT release_id FROM surgery s WHERE s.release_id = ar.id);
+
+---- Test results
+-- SELECT * FROM animal_release ar
+-- WHERE NOT EXISTS (SELECT release_id FROM surgery s WHERE s.release_id = ar.id);
+
+-- 2. When capture_location is null, assign value of release_location
+UPDATE animal_release
+SET capture_location = release_location
+WHERE capture_location IS NULL;
```

Comment:
I wasn't referring to changing the data via the gui. I meant in the http generating template - just filling in the value displayed to the end user from either the capture_location or release_location field depending on which one is not-null. And perhaps appending a star or something to indicate an ambiguity.

I don't know if that's appropriate for the data-model or not. 

It just seems weird to me, to turn null-values into observation values. 

For instance, how can we know that it's not null because the person recording didn't have that information rather than it always being the case that capture and release locations coincide when one of the fields is null? 

These kinds of changes really feels like QC correction for which there should be some type of record.